src/app/components/RemoveBtn.tsx

```diff
@@ -0,0 +1,27 @@
+"use client";
+
+import { HiOutlineTrash } from "react-icons/hi";
+import { useRouter } from "next/navigation";
+
+export default function RemoveBtn({ id }:any) {
+  const router = useRouter();
+  const removePost = async () => {
+    const confirmed = confirm("Are you sure?");
+
+    if (confirmed) {
+      const res = await fetch(`/api/items?id=${id}`, {
+        method: "DELETE",
+      });
+
+      if (res.ok) {
+        router.refresh();
```

Comment:
if we use state management combined with useEffect, updating data will be more discreet, it will not be necessary to update the entire page.